config.json.dist

```diff
@@ -1,3 +1,71 @@
+{
+  "pivotal": {
+    "baseUri": "https://www.pivotaltracker.com/services/v5",
+    "credentials": {
+      "owner": {
+        "token": ""
+      }
+    }
+  },
+  "trello": {
+    "baseUri": "https://api.trello.com/1",
+    "credentials": {
+      "owner": {
+        "key": "",
+        "token": ""
+      }
+    }
+  },
+  "todoist": {
+    "baseUri": "https://api.todoist.com/rest/v1",
+    "credentials": {
+      "owner": {
+        "token": ""
+      }
+    }
+  }
+}
+=======
+{
+  "pivotal": {
+    "baseUri": "https://www.pivotaltracker.com/services/v5",
+    "credentials": {
+      "owner": {
+        "token": ""
+      }
+    }
+  },
+  "trello": {
+    "baseUri": "https://api.trello.com/1",
+    "credentials": {
+      "owner": {
+        "key": "",
+        "token": ""
+      }
+    }
+  },
+  "todoist": {
+    "baseUri": "https://api.todoist.com/rest/v1",
+    "credentials": {
+      "owner": {
+        "token": ""
+      }
+    }
+  },
+  "sfdc": {
+    "baseUri": "/services/data/v48.0/sobjects",
+    "credentials": {
+      "owner": {
+        "clientId": "",
+        "clientSecret": "",
+        "securityToken": "",
+        "userName": "",
+        "password": "",
+      }
+    }
+  }
+}
+=======
```

Comment:
Please consider copying all the content from config.json.dist from develop branch to your branch in order to avoid conflicts.

Reply:
Also, please update your branch according to new changes in develop branch.